stories/components/STable.story.vue

```diff
@@ -85,35 +100,67 @@ const dropdownCreatedAt = createDropdown([
 const hasFilters = computed(() => {
   return [
     dropdownStatusSelected.value.length,
-    dropdownTypeSelected.value.length
+    dropdownTypeSelected.value.length,
+    dropdownTagsSelected.value.length
   ].some((length) => length)
 })
 
 const data = [
-  { name: 'Artwork 001', link: 'https://example.com', status: 'Published', type: 'Photo', createdAt: '2022-10-10' },
-  { name: 'Artwork 002', link: 'https://example.com', status: 'Draft', type: 'Icon', createdAt: '2022-10-09' },
-  { name: 'Artwork 003', link: 'https://example.com', status: 'Published', type: 'Photo', createdAt: '2022-10-02' },
-  { name: 'Artwork 004', link: 'https://example.com', status: 'Published', type: 'Illustration', createdAt: '2022-09-12' },
-  { name: 'Artwork 005', link: 'https://example.com', status: 'Archived', type: 'Other', createdAt: '2022-09-08' }
+  {
+    name: 'Artwork 001',
+    link: 'https://example.com',
+    status: 'Published',
+    type: 'Photo',
+    createdAt: '2022-10-10',
+    tags: ['Info', 'News', 'Latest']
+  },
+  {
+    name: 'Artwork 002',
+    link: 'https://example.com',
+    status: 'Draft',
+    type: 'Icon',
+    createdAt: '2022-10-09',
+    tags: ['Info']
+  },
+  {
+    name: 'Artwork 003',
+    link: 'https://example.com',
+    status: 'Published',
+    type: 'Photo',
+    createdAt: '2022-10-02',
+    tags: ['Info', 'News']
+  },
+  {
+    name: 'Artwork 004',
+    link: 'https://example.com',
+    status: 'Published',
+    type: 'Illustration',
+    createdAt: '2022-09-12',
+    tags: ['Info', 'News']
+  },
+  {
+    name: 'Artwork 005',
+    link: 'https://example.com',
+    status: 'Archived',
+    type: 'Other',
+    createdAt: '2022-09-08',
+    tags: ['Info']
+  }
 ]
 
 const filteredData = computed(() => {
   return data
     .filter((i) => filterBy(i.status, dropdownStatusSelected.value))
     .filter((i) => filterBy(i.type, dropdownTypeSelected.value))
+    .filter((i) => i.tags.some((tag) => filterBy(tag, dropdownTagsSelected.value)))
```

Comment:
Might wanna modify this. I'm not completely sure what's the expected behavior is with filtering multiple items.

Reply:
Ah, well this stuff depends on the business logic anyway so... Let's come back if we find issues on it 👀